hack/check/tools/go.mod

```diff
@@ -171,3 +171,5 @@ require (
 	mvdan.cc/lint v0.0.0-20170908181259-adc824a0674b // indirect
 	mvdan.cc/unparam v0.0.0-20220706161116-678bad134442 // indirect
 )
+
+replace github.com/gorilla/websocket => github.com/dvonthenen/websocket v1.5.1-dyv.2
```

Comment:
Just a note for the future - do we want to move the websocket repo to the deepgram org so it isn't all on you to maintain it?

Reply:
I have been maintaining the project since the little debacle with Gorilla jumping back in to support it again. I am more than happy to make a fork in the Deepgram org or evaluate the difference between what is there upstream.

Reply:
Will do this in the next release.